From 00897a058f591c1107a7c55f5fcbad4e5e46f8d0 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 18 Nov 2021 16:56:37 +0800
Subject: [PATCH 01/64] Revert "drm/msm/dsi: fix off by one in
 dsi_bus_clk_enable error handling"

This reverts commit c6b2400095ba232b1f2a0a499c2d7847940a9a2e.
---
 drivers/gpu/drm/msm/dsi/dsi_host.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/dsi/dsi_host.c b/drivers/gpu/drm/msm/dsi/dsi_host.c
index 96b5dcf8e454..b17ac6c27554 100644
--- a/drivers/gpu/drm/msm/dsi/dsi_host.c
+++ b/drivers/gpu/drm/msm/dsi/dsi_host.c
@@ -464,7 +464,7 @@ static int dsi_bus_clk_enable(struct msm_dsi_host *msm_host)
 
 	return 0;
 err:
-	while (--i >= 0)
+	for (; i > 0; i--)
 		clk_disable_unprepare(msm_host->bus_clks[i]);
 
 	return ret;
-- 
2.27.0

